screenpy/actions/silently.py

```diff
@@ -157,11 +47,46 @@ def Silently(duck: T_duck) -> Union[T_duck, T_silent_duck]:
     if settings.UNABRIDGED_NARRATION:
         return duck
 
+    # mypy really doesn't like monkeypatching
+    # See https://github.com/python/mypy/issues/2427
+
     if isinstance(duck, Performable):
-        return SilentlyPerformable(duck)
+        original_perform_as = duck.perform_as
+
+        def perform_as(self: Performable, actor: Actor) -> None:  # noqa: ARG001
+            """Direct the Actor to perform silently."""
+            with the_narrator.mic_cable_kinked():
+                original_perform_as(actor)
+                if not settings.UNABRIDGED_NARRATION:
+                    the_narrator.clear_backup()
+                return
+
+        duck.perform_as = types.MethodType(perform_as, duck)  # type: ignore[method-assign]
+
```

Comment:
I was going to ask if we should only do the monkeypatch on the duck (oy vey we've got a whole zoo over here) if `UNABRIDGED_NARRATION` were `False`, but i realized as i was typing up that comment that doing it the way you've got here handles when the settings change between the wrapping and the calling. 👍